### ISSUE TITLE
Have you ever got rejected in interview?

### ISSUE THREAD

**Issue by reachtokish** [tanaypratap/ama#248]
Hi Tanay, have you ever got rejected in an interview? if so what are those company names and how was your experience and for what reason they rejected you? What is the advice you want to give who gets rejected?

**Comment by tanaypratap** [comment]
Yes, I have been rejected multiple times. The ratio is not that high though. It would be a good thing to talk about. I don't apply to a company where there's high rate of rejection. 
Will talk about it at length in some podcast for sure. Thanks for the question.
